velox/dwio/parquet/writer/Writer.h

```diff
@@ -30,16 +31,40 @@ class ArrowDataBufferSink;
 
 struct ArrowContext;
 
+class DefaultFlushPolicy : public dwio::common::FlushPolicy {
+ public:
+  DefaultFlushPolicy(uint64_t rowsInRowGroup, int64_t bytesInRowGroup)
+      : rowsInRowGroup_(rowsInRowGroup), bytesInRowGroup_(bytesInRowGroup) {}
+
+  bool shouldFlush(
+      const dwio::common::StripeProgress& stripeProgress) override {
+    return stripeProgress.stripeRowCount >= rowsInRowGroup_ ||
+        stripeProgress.stripeSizeEstimate >= bytesInRowGroup_;
+  }
+
+  void onClose() override {
+    // No-op
+  }
+
+ private:
+  const uint64_t rowsInRowGroup_;
+  const int64_t bytesInRowGroup_;
+};
+
 struct WriterOptions {
   bool enableDictionary = true;
   int64_t dataPageSize = 1'024 * 1'024;
-  int32_t rowsInRowGroup = 10'000;
+  uint64_t rowsInRowGroup = 10'000;
```

Comment:
Can we remove this two and use the `flushPolicyFactory` only (you can create the `DefaultFlushPolicy` with default parameters if it is not set)

Reply:
sure, removed these variable